.github/workflows/regallint.yml

```diff
@@ -9,4 +9,4 @@ jobs:
         uses: StyraInc/setup-regal@v0.2.0
         with:
           version: v0.11.0
-      - run: regal lint --format=github cvat/apps/iam/rules
+      - run: regal lint --format=github cvat/apps/*/rules
```

Comment:
Do you think we can update regal in the PR? https://github.com/StyraInc/regal/releases/tag/v0.21.3

Reply:
I ended up doing that in a separate PR: #7824.